plugins/plugin-chart-echarts/src/MixedTimeseries/EchartsMixedTimeseries.tsx

```diff
@@ -95,12 +95,12 @@ export default function EchartsMixedTimeseries({
         handleChange([seriesName], seriesIndex);
       }
     },
-    mousemove: params => {
-      currentSeries.name = params.seriesName;
-    },
     mouseout: () => {
       currentSeries.name = '';
     },
+    mouseover: params => {
+      currentSeries.name = params.seriesName;
+    },
```

Comment:
Move to use the new `mouseover` event introduced for line charts in ECharts 5.2.2